influxd.yaml

```diff
@@ -0,0 +1,41 @@
+package:
+    name: influxd
```

Comment:
should this be `influxdb`?

Reply:
Strangely the daemon binary is actually called influxd: https://github.com/influxdata/influxdb/tree/master/cmd/influxd